spec/serializers/api/application_serializer_spec.rb

```diff
@@ -256,6 +256,32 @@
       it "serializes the `schedule_identifier`" do
         expect(attributes["schedule_identifier"]).to eq(application.schedule.identifier)
       end
+
+      context "when LP_SELF_SERVE feature flag is on" do
+        let(:application) { build(:application, :senco, cohort:, course:, private_childcare_provider:, itt_provider:, school:) }
+
+        before do
+          Flipper.enable(Feature::LP_SELF_SERVE)
+        end
+
+        it "serializes the `works_as_senco`" do
+          expect(attributes["works_as_senco"]).to eq(application.senco_in_role)
+        end
+
+        it "serializes the `senco_start_date`" do
+          expect(attributes["senco_start_date"]).to eq(application.senco_start_date.to_json)
```

Comment:
we probably need to test a non-senco application with the feature flag on - to check `attributes["senco_start_date"] == nil`